Headers/Public/VLCMedia.h

```diff
@@ -217,7 +217,7 @@ typedef NS_ENUM(NSUInteger, VLCMediaType) {
  * equivalent in lexical value, and NSOrderedDescending if the URL of the
  * receiver follows media. If media is nil, returns NSOrderedDescending.
  */
-- (NSComparisonResult)compare:(VLCMedia *)media;
+- (NSComparisonResult)compare:(nullable VLCMedia *)media;
```

Comment:
In [VLCMedia.m::compare](https://github.com/mkchoi212/vlckit/blob/4ae19df050220cdfec5bf3e2998acb6246d2f0ae/Sources/VLCMedia.m#L299), the line
```
if (!media)
```
can not be satisfied unless the parameter is `nullable`.
Or can it without it??